hieradata/common.yaml

```diff
@@ -416,6 +416,7 @@ govuk::apps::content_publisher::db_hostname: "postgresql-primary-1.backend"
 govuk::apps::content_publisher::db_port: 6432
 govuk::apps::content_publisher::db_allow_prepared_statements: false
 govuk::apps::content_publisher::db::backend_ip_range: "%{hiera('environment_ip_prefix')}.3.0/24"
+govuk::apps::content_publisher::google_tag_manager_id: "GTM-NQXC4TG"
```

Comment:
I just chatted with @alex-ju and it seems we only need this id to send things to google tag manager. On that basis I think we should move this out of common so that we don't use this when apps are running in dev vm.